amundsen_application/static/js/components/SearchPage/SearchBar/index.tsx

```diff
@@ -60,7 +60,10 @@ export class SearchBar extends React.Component<SearchBarProps, SearchBarState> {
   handleValueSubmit = (event: React.FormEvent<HTMLFormElement>) : void => {
     event.preventDefault();
     if (this.isFormValid()) {
-      const pathName = `/search?searchTerm=${this.state.searchTerm}&selectedTab=table&pageIndex=0`;
+      let pathName = '/';
+      if (this.state.searchTerm !== '') {
```

Comment:
Does this account for cases where a blank space is passed to the search bar? Not sure if there is a `trim()` somewhere along the way.

Reply:
no it does not, empty spaces returns with: "Your search - - did not match any tables result". Do we want the behavior to be that or should the user be redirected back to home if the bar is all white space?